frontend/src/data/variables/BrfssProvider.ts

```diff
@@ -98,8 +96,26 @@ class BrfssProvider extends VariableProvider {
           estimated_total_diabetes: (series) => series.sum(),
         })
         .resetIndex();
+
+      // We need to get the national acs dataset here in order to
+      // get the national pct share of population for each state.
+      acsBreakdowns.geography = "national";
+      const acsQueryResponse = await this.acsProvider.getData(
+        new MetricQuery(["population_pct"], acsBreakdowns)
+      );
+      consumedDatasetIds = consumedDatasetIds.concat(
+        acsQueryResponse.consumedDatasetIds
+      );
+
+      const acsPopulation = new DataFrame(acsQueryResponse.data);
```

Comment:
nit: since acsPopulation is only used once, i might just inline it in the joinOnCols(). If you didn't want to inline it I would at least remove the line break between them

I might also rename the two acsPopulations to be specific to the breakdowns that queried for them as well